server/server_test.go

```diff
@@ -8812,9 +8812,13 @@ func TestServer_ListServiceTests(t *testing.T) {
 				Name: "kafka",
 				Options: client.ServiceTestOptions{
 					"cluster": "example",
-					"topic":   "test",
 					"key":     "key",
 					"message": "test kafka message",
+					"target": map[string]interface{}{
+						"PartitionAlgorithm": "",
+						"PartitionById":      true,
+						"Topic":              "test",
+					},
```

Comment:
@docmerlin will this schema change break people when they upgrade? I could rework it so topic stays a top-level key, and the partitioning info is in its own struct